spec/jobs/document_proofing_job_spec.rb

```diff
@@ -335,5 +336,31 @@
         expect { perform }.to raise_error(JobHelpers::StaleJobHelper::StaleJobError)
       end
     end
+
+    context 'with data url body' do
+      let(:body) { DocAuthImageFixtures.document_front_image_data_uri }
+
+      it 'decrypts the image correctly' do
+        expect_any_instance_of(DocAuth::Mock::DocAuthMockClient).
+          to receive(:post_images).
+          with(hash_including(front_image: DocAuthImageFixtures.document_front_image.b)).
+          and_call_original
+
+        perform
+      end
+    end
+
+    context 'with jpg file body' do
+      let(:body) { DocAuthImageFixtures.document_front_image }
+
+      it 'decrypts the image correctly' do
+        expect_any_instance_of(DocAuth::Mock::DocAuthMockClient).
+          to receive(:post_images).
+          with(hash_including(front_image: DocAuthImageFixtures.document_front_image.b)).
+          and_call_original
+
+        perform
+      end
+    end
```

Comment:
The fact that the build passes despite the fact that `Idv::DataUrlImage::InvalidUrlFormatError` does not exist in the current state of this branch is a bit alarming 😅 Can we add coverage for the thrown error?

```suggestion
    end

    context 'with invalid data url body' do
      let(:body) { 'data:"' }

      it 'gracefully degrades' do
        expect_any_instance_of(DocAuth::Mock::DocAuthMockClient).
          to receive(:post_images).
          with(hash_including(front_image: nil)).
          and_call_original

        perform
      end
    end
```

(You'll definitely need to rebase / merge main this time)